Library/Formula/libspectre.rb

```diff
@@ -0,0 +1,36 @@
+class Libspectre < Formula
+  desc "Small library for rendering Postscript documents"
+  homepage "https://wiki.freedesktop.org/www/Software/libspectre/"
+  url "http://libspectre.freedesktop.org/releases/libspectre-0.2.7.tar.gz"
+  sha256 "e81b822a106beed14cf0fec70f1b890c690c2ffa150fa2eee41dc26518a6c3ec"
+
+  depends_on "ghostscript"
+
+  def install
+    ENV.append "CFLAGS", "-I#{Formula["ghostscript"].opt_include}/ghostscript"
```

Comment:
Ah, alright. I'll take it as is then. If you're feeling kind it might be cool to ask upstream to work in a `--with-gs=$DIR` option or similar.